app/Http/Controllers/Buyer/BuyerController.php

```diff
@@ -17,4 +17,9 @@ public function reviewPage()
     {
         return view('buyers.reviewpage');
     }
+
+        public function service()
+    {
```

Comment:
Can you fix the line indention of line 21?

Reply:
I fixed the line indention of line 21 in [1c2d8fd](https://github.com/kredo-school/16th_rankshty/pull/17/commits/1c2d8fdc9a9942af9f8a9747ac774e96c1119e2c)